dev/jenkins/build.sh

```diff
@@ -68,8 +68,14 @@ if [ "$RUN_MAVEN" == "true" ]; then
   then
     ALLUXIO_BUILD_FORKCOUNT=4
   fi
-
-  mvn -Duser.home=/home/jenkins -T 4C clean install -Pdeveloper -Dmaven.javadoc.skip -Dsurefire.forkCount=${ALLUXIO_BUILD_FORKCOUNT} ${mvn_args} $@
+  JAVA_HOME_BACKUP=${JAVA_HOME}
```

Comment:
Can you add a comment that mentions that these default env variables are pointing to java 11 initially?

Reply:
they might. This file is shared between java 8 and java 11 tests. in both cases, we use java 8 to compile and use the image default (java8 or java 11 ) to run the test

Reply:
i will add a comment anyways